adapters/utils/networkUtils.js

```diff
@@ -140,13 +130,50 @@ const parseDestResponse = (destResponse, destination) => {
   }
   return {
     responseBody: parsedDestResponseBody,
-    status
+    status,
+    payload: destResponse.payload
+  };
+};
+
+// Function to process wrapped axioss response from internal http client compatible for response handlers
+const processAxiosResponse = clientResponse => {
+  if (!clientResponse.success) {
+    const { response, code } = clientResponse.response;
+    // node internal http client failure cases
+    if (!response && code) {
+      const nodeClientError = nodeSysErrorToStatus(code);
+      return {
+        response: nodeClientError.message,
+        status: nodeClientError.status
+      };
+      return processedResponse;
```

Comment:
this line can be removed. same for L158.

Reply:
Done